src/components/modals/LogsModal.tsx

```diff
@@ -83,29 +83,31 @@ export const LogsModal = (props: Props) => {
     bridge.resolve()
   }
 
-  const handleSend = async () => {
+  const handleSend = async (): Promise<void> => {
     logs.info.userMessage = userMessage
     logs.activity.userMessage = userMessage
 
     const underDuress = account.isDuressAccount
 
     await Promise.all([
-      sendLogs(logs.activity, underDuress).catch((e: any) => {
+      sendLogs(logs.activity, underDuress).catch((error: unknown) => {
+        const message = error instanceof Error ? error.message : String(error)
```

Comment:
`String(error)` should result in `Error: ${error.message}` if its an instance of Error. I can see why you wouldn't want the `"Error: "` prefix, but I can also see why you might want it.